react/week4/github-search/src/components/RenderUsers.js

```diff
@@ -0,0 +1,23 @@
+import React, { useContext } from "react";
+import { userContext } from "../userContext";
+
+const RenderUsers = () => {
+  const value = useContext(userContext);
+  const users = value.users;
+  return (
+    <div>
+      {value.loading && <p>Loading...</p>}
+      {users.length === 0 ? (
```

Comment:
Hi @GreeshmaRahool, there is an error on line 10 when trying to clear the input field, try to debug it.